docker/edge/http/pep-config.js

```diff
@@ -0,0 +1,81 @@
+const config = {};
+
+function toBoolean(env, defaultValue) {
+  return env !== undefined ? env.toLowerCase() === 'true' : defaultValue;
+}
+
+function to_array(env, default_value){
+    return (env !== undefined) ? env.split(',') : default_value;
+}
+
+// Used only if https is disabled
+config.pep_port = process.env.PEP_PROXY_PORT || 5556;
+
+// Set this var to undefined if you don't want the server to listen on HTTPS
+config.https = {
+  enabled: toBoolean(process.env.PEP_PROXY_HTTPS_ENABLED, false),
+  cert_file: 'cert/cert.crt',
+  key_file: 'cert/key.key',
+  port: process.env.PEP_PROXY_HTTPS_PORT || 443,
+};
+
+//IDM configration
+config.idm = {
+  host: process.env.PEP_PROXY_IDM_HOST || '180.179.214.199',
+  port: process.env.PEP_PROXY_IDM_PORT || 3000,
+  ssl: toBoolean(process.env.PEP_PROXY_IDM_SSL_ENABLED, false),
+};
+
+//Apllication information, for which account has been registered on IDM
+config.app = {
+  host: process.env.PEP_PROXY_APP_HOST || '180.179.214.211',
+  port: process.env.PEP_PROXY_APP_PORT || '8060',
+  ssl: toBoolean(process.env.PEP_PROXY_APP_SSL_ENABLED, false), // Use true if the app server listens in https
+};
+
+config.organizations = {
+  enabled: toBoolean(process.env.PEP_PROXY_ORG_ENABLED, false),
+  header: process.env.PEP_PROXY_ORG_HEADER || 'fiware-service'
+}
+
+// Credentials obtained when registering PEP Proxy in app_id in Account Portal
+config.pep = {
+  app_id: process.env.PEP_PROXY_APP_ID || '6e396def-3fa9-4ff9-84eb-266c13e93964',
```

Comment:
[Suggestion] Hardcoded code must be replaced with configurable one.

Reply:
It is not hard coded, rather updated while  bringing the up the edge